vendor/engines/split_accounts/app/models/split_accounts/reports/export_raw_transformer.rb

```diff
@@ -1,12 +1,10 @@
-require "hash_helper"
 module SplitAccounts
   module Reports
     class ExportRawTransformer
-      include HashHelper
       include ActionView::Helpers::NumberHelper
 
       def transform(original_hash)
-        insert_into_hash_after(original_hash, :actual_total, split_percent: method(:split_percent))
+        original_hash.merge(split_percent: method(:split_percent))
       end
 
```

Comment:
Yeah, that's probably a good idea to keep consistency if we add new columns later.